### PR TITLE
feat: support command for minioJob

### PR DESCRIPTION
**Pull request by jiuker**
feat: support command for minioJob
how to test:
```yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: mc-job-sa
  namespace: minio-operator
---
apiVersion: sts.min.io/v1alpha1
kind: PolicyBinding
metadata:
  name: mc-job-bingding
  namespace: minio-operator
spec:
  application:
    serviceaccount: mc-job-sa
    namespace: minio-operator
  policies:
    - consoleAdmin
---
apiVersion: job.min.io/v1alpha1
kind: MinIOJob
metadata:
  name: minio-test-job
  namespace: minio-operator
spec:
  serviceAccountName: mc-job-sa
  tenant:
    name: myminio
    namespace: minio-operator
  commands:
    - op: make-bucket
      args:
        name: memes
    - op: stat
      command:
        - "mc"
        - "stat"
        - "myminio/memes"
        - "--insecure"
```

### CODE REVIEW

**Review by cniackz — APPROVED** [review]
I love this feature

File: pkg/utils/miniojob/types.go [hunk 1]
```diff
@@ -131,16 +131,20 @@ func (jobCommand *MinIOIntervalJobCommand) createJob(ctx context.Context, k8sCli
...
-	for _, command := range commands {
-		trimCommand := strings.TrimSpace(command)
-		if trimCommand != "" {
-			jobCommands = append(jobCommands, trimCommand)
+	if len(jobCommand.CommandSpec.Command) == 0 {
+		commands := []string{"mc"}
+		commands = append(commands, strings.SplitN(jobCommand.MCOperation, "/", -1)...)
+		commands = append(commands, strings.SplitN(jobCommand.Command, " ", -1)...)
+		for _, command := range commands {
+			trimCommand := strings.TrimSpace(command)
```
Comment (shtripat) [inline pkg/utils/miniojob/types.go:139]:
```suggestion
			trimmedCommand := strings.TrimSpace(command)
```

**Review by shtripat — APPROVED** [review]
LGTM. Verified.
One suggestion.

**Comment by ramondeklein** [comment]
I'm fine with the code, but isn't it a security risk to allow running commands as any service account? If you know the name of a privileged service account, then you can run any command using that account.

**Review by ramondeklein — APPROVED** [review]

**Review by pjuarezd — APPROVED** [review]

**Comment by ramondeklein** [comment]
> I'm fine with the code, but isn't it a security risk to allow running commands as any service account? If you know the name of a privileged service account, then you can run any command using that account (if you're entitled to submit Minio jobs).

@dvaldivia @harshavardhana What's your opinion? You can also submit jobs that issue `wget` or `curl` commands this way. Not sure if we would want that? I know our Docker images have only a few tools installed, but running any tool under someone else's credentials feels too tricky to me...

**Pull request merged**